app/src/main/java/org/mozilla/focus/fragment/BrowserFragment.java

```diff
@@ -802,16 +822,16 @@ public void onClick(View view) {
                 break;
 
             case R.id.add_to_homescreen:
-                final IWebView webView = getWebView();
-                if (webView == null) {
-                    return;
-                }
-
-                final Bitmap icon = webView.getIcon();
-                final String url = webView.getUrl();
-                final String title = webView.getTitle();
+                if (!isLoading()) {
```

Comment:
@antlam changed this, and now we will grey out the menu item for add to homescreen if the user opens the menu while the page is loading.